### ISSUE TITLE
Please add support for HmIP-eTRV-CL

### ISSUE THREAD

**Issue by bliblabl8** [hahn-th/homematicip-rest-api#493]
Please add support for HmIP-eTRV-CL

[hmip-config_AE72.json.txt](https://github.com/hahn-th/homematicip-rest-api/files/11876758/hmip-config_AE72.json.txt)

**Comment by bliblabl8** [comment]
Is the support added in the HA integration already? Temperature sensor still shows "N/A" for HmIP-eTRV-CL. Do I have to reload the integration or delete the device to get it back then with a working temperature sensor?

**Comment by bliblabl8** [comment]
Is the support added in the HA integration already? Temperature sensors is working for this device but the entity "climate.schlafzimmer" for this device isn't working (shown as "not available" in HA)

**Comment by hahn-th** [comment]
Hi. The device is in a "HEATING" Group. This should normally be enough to make the climate work. Is it only Schlafzimmer, which is not available in HomeAssistant?

**Comment by bliblabl8** [comment]
yes. its only this one entity, thats unavailable, three others are working fine.

**Comment by hahn-th** [comment]
i am gonna check it

**Comment by bliblabl8** [comment]
Now the device is working. No clue wether update to 2024.7.1 changed something or just the reboot of HA fixed it. But it's working fine now. Issue can be closed. Maybe you want to update the GitHub integration site that this device is supported.
